nautobot/docs/configuration/optional-settings.md

```diff
@@ -257,6 +257,24 @@ The global Celery task hard timeout (in seconds). Any background task that excee
 
 ---
 
+## CELERY_WORKER_PROMETHEUS_PORT
+
++++ 1.5.x
+
+Default: `` (disabled)
+
+Environment Variable: `NAUTOBOT_CELERY_WORKER_PROMETHEUS_PORT`
+
+Ports for prometheus metric HTTP server running on the celery worker.
+
+Normally this should be set to a single port, unless you have multiple workers running on a single machine, i.e.
+sharing the same available ports. In that case you need to specify a range of ports greater than or equal to the
+highest amount of workers you are running on a single machine (comma-separated, like "8080,8081,8082"). You can then use
+the `target_limit` parameter to the Prometheus `scrape_config` to ensure you are not getting duplicate metrics in that
+case. Set this to an empty string to disable it.
```

Comment:
```suggestion
case. Set this to an empty list to disable it.
```

Reply:
Wouldn't it be an empty string still in this case, as in `""`? I realize this maps to an empty list through the configuration mechanism, but the actual setting is still a string type, right?

Reply:
If I'm reading right, `CELERY_WORKER_PROMETHEUS_PORTS` should be an empty list; to make this happen with the built-in environment variables, you would set `NAUTOBOT_CELERY_WORKER_PROMETHEUS_PORTS` to an empty string?

Reply:
Ahh right, I was mixing up environment variables and the actual settings. Fixed it, thanks!